src/Symfony/Component/Pgp/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/pgp",
+    "type": "library",
+    "description": "Allows encryption of MIME messages via OpenPGP",
+    "keywords": ["openpgp", "pgp", "mime"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/mime": "^7.2",
+        "pear/crypt_gpg": "^1.6"
```

Comment:
The [`pear/crypt_gpg` package](https://github.com/pear/Crypt_GPG) is actively maintained with support for old and newest PHP versions. It seems that we can rely on it and trust it.